scripts/dashboard/install-cephadm-e2e-deps.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+set -ex
+
+on_error() {
+    if [ "$1" != "0" ]; then
+        printf "\n\nERROR $1 thrown on line $2\n\n"
+        printf "\n\nCollecting info...\n\n"
+        sudo journalctl --since "10 min ago" --no-tail --no-pager -x
+        printf "\n\nERROR: displaying podman logs:\n\n"
+        podman logs -l
+        printf "\n\nTEST FAILED.\n\n"
+    fi
+}
+
+trap 'on_error $? $LINENO' ERR
+
+sudo apt -y install libvirt-daemon-system libvirt-daemon-driver-qemu qemu-kvm libvirt-clients
+
+sudo usermod -aG libvirt $(id -un)
+newgrp libvirt  # Avoid having to log out and log in for group addition to take effect.
+sudo systemctl enable --now libvirtd
+
+# Add podman repo for ubuntu focal.
+VERSION_ID='20.04'
+sudo sh -c "echo 'deb http://download.opensuse.org/repositories/devel:/kubic:/libcontainers:/stable/xUbuntu_${VERSION_ID}/ /' > /etc/apt/sources.list.d/devel:kubic:libcontainers:stable.list"
+wget -nv https://download.opensuse.org/repositories/devel:kubic:libcontainers:stable/xUbuntu_${VERSION_ID}/Release.key -O- | sudo apt-key add -
+sudo apt update -y
+
+# Install required deps.
+sudo apt install -y nodejs npm openssh-server podman
+
+mkdir -p ~/.kcli ~/.ssh
+if [[ ! -f "$HOME/.ssh/id_rsa" ]]; then
+    ssh-keygen -t rsa -q -f "$HOME/.ssh/id_rsa" -N ""
```

Comment:
Please name it something unique to this test

Reply:
Maybe symlink to it inside the container temporarily or something.

Reply:
Or better yet, write the key to `$WORKSPACE/id_rsa` and mount that inside the container at `~/.ssh`

Reply:
The keys are not only used by podman but by KCLI, the VM manager, so the VMs are using it so form the host you can SSH to any VM. I tried with a dedicated key and I was not able to SSH into any VM.
Nevertheless I'll investigate this a bit more.

Reply:
@djgalloway Now we store the key in KCLi config folder:
```
KCLI_CONFIG_DIR="${HOME}/.kcli"
mkdir -p ${KCLI_CONFIG_DIR}
if [[ ! -f "${KCLI_CONFIG_DIR}/id_rsa" ]]; then
    ssh-keygen -t rsa -q -f "${KCLI_CONFIG_DIR}/id_rsa" -N ""
fi
```
Let us know if this works for you.